src/diffpy/structure/atom.py

```diff
@@ -224,9 +310,34 @@ def xyz_cartn(self, value):
     # anisotropy
 
     def _get_anisotropy(self):
+        """
+        Obtain the anisotropic thermal displacements parameters ``anisotropy``.
+
+        Parameters
+        ----------
+        None.
+
+        Returns
+        -------
+        self._anisotropy : Boolean
+            flag for anisotropic thermal displacements parameters. 
+        """
+
         return self._anisotropy
 
     def _set_anisotropy(self, value):
+        """Set the anisotropic thermal displacements parameters ``anisotropy``.
+
+        Parameters
+        ----------
+        value: Boolean
+            The flag for anisotropic thermal displacements parameters to be set.
+
+        Returns
+        -------
+        None.
+        """
+
```

Comment:
`_get_anisotropy` and `_set_anisotropy` are used in an old-style definition of `anisotropic` property at line 351.  The  `doc` argument there hides both docstrings.

Please merge `_set_anisotropy` docstring into the `_get_anisotropy` one and reformat as per sphinx-napoleon property format.  After that remove the `doc` argument in the property call later.  Make sure that a correct docstring is displayed by
```
pydoc diffpy.structure.Atom.anisotropy
```

Do the same fix for `U`, `Uisoequive`, etc. properties below.